diff --git a/dist/index.js b/dist/index.js
index ba566d95956789a83721f4b36796325c603a7923..9212fc95331e0ace8883a757839a798f8c280c57 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -174,8 +174,8 @@ export default function netlifyIntegration(integrationConfig) {
 				const ctx = createContext({
 					request,
 					params: {}
+                    locals: { context }
 				});
-				ctx.locals = { netlify: { context } }
 				// https://docs.netlify.com/edge-functions/api/#return-a-rewrite
 				ctx.rewrite = (target) => {
 					if(target instanceof Request) {
@@ -197,7 +197,10 @@ export default function netlifyIntegration(integrationConfig) {
 				};
 				const next = () => {
 					const { netlify, ...otherLocals } = ctx.locals;
+                    console.log('OTHER LOCALS FROM NETLIFY EDGE RUNNER 1:', {...otherLocals});
+                    console.log('OTHER LOCALS FROM NETLIFY EDGE RUNNER 2:', ctx.locals);
 					request.headers.set("x-astro-locals", trySerializeLocals(otherLocals));
+                    console.log('CUSTOM LOCALS HEADER SET BY NETLIFY EDGE RUNNER:', request.headers.get("x-astro-locals"));
 					request.headers.set("x-astro-middleware-secret", "${middlewareSecret}");
 					return context.next();
 				};
